src/en/holymanga/src/eu/kanade/tachiyomi/extension/en/holymanga/HolyManga.kt

```diff
@@ -2,4 +2,10 @@ package eu.kanade.tachiyomi.extension.en.holymanga
 
 import eu.kanade.tachiyomi.multisrc.zbulu.Zbulu
 
-class HolyManga : Zbulu("HolyManga", "https://w33.holymanga.net", "en")
+class HolyManga : Zbulu(
+    "HolyManga",
+    "https://w34.holymanga.net",
+    "en",
+) {
+    override val supportsLatest = false
```

Comment:
Is this correct? `/latest-update/` seems to work on the site.

Reply:
> Is this correct? `/latest-update/` seems to work on the site.

The [`/latest-update/`](https://w34.holymanga.net/latest-update/) and [`/manga-list/`](https://w34.holymanga.net/manga-list) contain the same content

Reply:
So `/latest-update/` and `/manga-list/` is the same, they both look like they should be the latest tab. `/popular-manga/` is however different.